### DISCUSSION TITLE
Pytorch Baseline Implementation

### DISCUSSION THREAD

**Discussion by ethanchwang** [fwillett/speechBCI#14] [General]
Hi! We noticed that there was a "NPTL Pytorch Baseline (5gram + OPT6B)" on the leaderboard and we were wondering if this was an implementation by the organizing team. If so, would you be willing to share it? Thanks!

**Comment by fwillett** [comment]
Yep, here's our implementation that Chaofei Fan has posted on github:  GitHub
- cffan/neural_seq_decoder <https://github.com/cffan/neural_seq_decoder>.
Happy to help! How have things been going for you with the competition?
-Frank
